### ISSUE TITLE
I Submitted a FTS for resource_type=organization and results included credentials, learning opps, and transfer value profiles

### ISSUE THREAD

**Issue by mparsons-ce** [CredentialEngine/CredentialRegistry#429]
I was testing a registry import locally. I made a request for organizations. I found that near the end of the list of envelopes I started to see credential type envelopes, and then some learning opportunities. I did some additional queries and found some transfer value profiles were being returned even though the search specify resource_type=organization
I used the following query. I set the per_page to be higher, and the page to be equal to three as the issue seemed only occur at the end of the results.
https://sandbox.credentialengineregistry.org/ce-registry/search?**resource_type=organization**&page=3&per_page=120&from=2021-01-01T00:00:00&sort_by=updated_at&sort_order=dsc

![image](https://user-images.githubusercontent.com/58825866/115079232-8068d800-9ec6-11eb-95f8-6c836fc56309.png)

I had recently seen weird events in the sandbox where a learning opp was imported as an organization, or a credential. So this issue may be specific to an organization search.

**Comment by excelsior** [comment]
@mparsons-ce The way the FTS works is we look for resources matching the given criteria, then return the full envelope for each found resource. The problem is an envelope can contain resources of various types. For instance, the above envelope contains two top-level resources:

https://sandbox.credentialengineregistry.org/envelopes/7e366090-89a5-4b2b-b9d4-f3b37177a9d4

The first one is a Certificate and the last one is a QACredentialOrganization. The latter is returned by the FTS, but since the envelopes contains a Certificate as well, it gets returned, too.

In other words, it's an intended behavior. We can filter out the non-matching resources when building the response. Let me know if you'd prefer it that way.

**Comment by mparsons-ce** [comment]
per April 22, 2021 meeting
Suggested approaches
- To be able to specify the envelope type (envelope_ctdl_type) as a search parameter. 
- Or only return results where the envelope_ctdl_type is the same as the type of the first node in the graph and the latter matches the resource_type requested in the search

In both cases we would want to be able to specify the alias. That is to say: envelope_ctdl_type=credential and not have to list the specific type of ceterms:Certificate, ceterms:Certification, etc.

**Comment by excelsior** [comment]
@mparsons-ce I fixed it using the second approach. It's available for testing on sandbox.

**Comment by mparsons-ce** [comment]
@excelsior I tested the organization search - no credentials are being returned. 
Please promote any necessary changes to staging and productions.
Thanks

**Comment by excelsior** [comment]
@mparsons-ce Done!
